lib/content-store/rest-handler.js

```diff
@@ -18,14 +18,14 @@ function transformImage(obj, sizes) {
   }
 
   const image = obj;
-  image.srcset = image.srcset || [];
+  image.props.srcset = image.props.srcset || [];
 
   sizes.forEach((size) => {
-    const imgString = `${image.id}/width-${size}/`;
+    const imgString = `${image.props.id}/width-${size}/`;
```

Comment:
It's currently an expectation as much as props. The image id is used to create the signature so that would also fail if it didn't exist.